webdriver-spec.html

```diff
@@ -2872,12 +2882,10 @@
   or for the <a>session page load timeout</a> milliseconds to pass,
   whichever occurs sooner.
 
-  <p class=issue>This doesn’t actually wait for the page to load
-   if traversing the <a>session history</a> kicks off a navigation.
-
- <li><p>If the previous step completed by the <a>session page load timeout</a> being reached,
-   and the browser is not currently displaying an alert,
-   return <a>error</a> with <a>error code</a> <a>timeout</a>.
+ <li><p>If the previous step completed by the <a>session page load
+   timeout</a> being reached, and the browser is not currently
```

Comment:
instead of `and the browser is not currently displaying an alert,` we could put something like `and <a data-lt='Handle any user prompts'>user prompts have been handled</a>`